fdt/items.py

```diff
@@ -342,8 +342,7 @@ def to_dtb(self, strings: str, pos: int = 0, version: int = Header.MAX_VERSION):
             strpos = len(strings)
             strings += self.name + '\0'
         blob = pack('>III', DTB_PROP, len(self.data) * 4, strpos)
-        for word in self.data:
-            blob += pack('>I', word)
+        blob += bytes().join([BIGENDIAN_WORD.pack(word) for word in self.data])
```

Comment:
This is the critical change - it makes the write operation complete in a small number of seconds rather than at least 10's of minutes.